site/docs/v1/tech/apis/_tenant-application-email-configuration.adoc

```diff
@@ -34,6 +37,9 @@ ifdef::is_tenant[]
 The host name of the SMTP server that FusionAuth will use.
 +
 Prior to version 1.28.0 this value was required.
+
+[field]#{base_field_name}.emailConfiguration.implicitEmailVerificationAllowed# {optional_tag} {default_true} [since]#Available since 1.32.0#::
+When set to true, this allows email to be verified as a result of completing a similar based email workflow such as change password. When set to false, the user must explicitly complete the email verification workflow even if the user has already completed a similar email workflow such as change password.
```

Comment:
Took most of this from the tooltip.  Should it be "...result of completing a similar **email based workflow**.." or as is?  I keep going back and forth...

Reply:
Yeah `email based` is probably better.